_episodes/12_ignoring_files.md

```diff
@@ -259,7 +259,7 @@ Your branch is ahead of 'origin/main' by 2 commits.
 nothing to commit, working tree clean
 ```
 
-Notice how we still have 2 some commits that have not been pushed to the remote
+Notice how we still have some commits that have not been pushed to the remote
 repository. We'll tidy up by pushing these commits, so that our local repository
 and remote repository are up-to-date with each other:
 
```

Comment:
Nit: This really is a nit, but wouldn't you need a push _and a pull_ to really make sure that the local and remote are up to date _with each other_? 
Also, not sure that _tidying up_ really describes this part of the process. I think it's more fundamental than that.
Suggestion: `We'll update the remote repository now:`

Reply:
Very good points - changed the wording around 'tidying', left it just as pushing for the time being for simplicity, but will keep it in mind when revisiting the material in the future!